Galaxus.Functional.Tests/(Either)/EitherTests.cs

```diff
@@ -47,134 +47,134 @@ public void UnionEqualityAndHashCodeWithMixedTypeVariants()
         {
             var union1 = new Either<string, int>(117);
             var union2 = new Either<string, int>("hello");
-            Assert.AreNotEqual(union1, union2);
+            Assert.AreNotEqual(expected: union1, actual: union2);
             Assert.AreNotEqual(union1.GetHashCode(), union2.GetHashCode());
         }
 
         [Test]
         public void UnionEqualityWithMixedValueTypeVariantsBothZero()
         {
-            var union1 = new Either<byte, int>(default(byte));
+            var union1 = new Either<byte, int>(default);
             var union2 = new Either<byte, int>(default(int));
-            Assert.AreNotEqual(union1, union2);
+            Assert.AreNotEqual(expected: union1, actual: union2);
         }
-        
+
         [Test]
         public void IsAIsBAreEvaluatedCorrectly()
         {
             var someA = new Either<string, int>("hello");
             var someB = new Either<string, int>(117);
 
-            Assert.IsTrue(someA.IsA);
-            Assert.IsFalse(someA.IsB);
-            Assert.IsFalse(someB.IsA);
-            Assert.IsTrue(someB.IsB);
+            Assert.IsTrue(condition: someA.IsA);
```

Comment:
Named arguments weren't explicitly configured in the editorconfig, I've left it as-is because it was "pleasingly consistent" or something like that.

You mentioning named arguments exactly on the single scenario where I hate them (only one argument) is reason enough for me to actually change it. I've configured it all to explicitly use positional instead of named arguments.

I'm not smart enought to configure it for "positional for one argument, named for >=2 arguments" (which I'd love to have). Maybe you wanna give it a try? :D